lib/shopify_api/rest/base.rb

```diff
@@ -16,6 +16,8 @@ class Base
       @paths = T.let([], T::Array[T::Hash[Symbol, T.any(T::Array[Symbol], String, Symbol)]])
       @custom_prefix = T.let(nil, T.nilable(String))
 
+      @aliased_properties = T.let({}, T::Hash[String, String])
```

Comment:
does it make sense to add a comment here illustrating the purpose of ` @aliased_properties`?

Reply:
Yes, I think so.